### ISSUE TITLE
Invalid XML (in SMIL) not coming up in ePubCheck

### ISSUE THREAD

**Issue by bdugas** [IDPF/epubcheck#701]
Hello,

I'm logging this issue we've recently come across at Kobo where files that contain invalid XML pass ePubCheck.

We had several ePubs that passed ePubCheck save for the flag “attribute "media-overlay" not allowed here; expected attribute "fallback", "fallback-style", "media-type", "required-modules" or "required-namespace" which my research indicates as erroneous.
 
When loaded into our CMS these files either fail to generate or will not open on our reading platforms. We discovered that this was the result of invalid XML in the SMIL. When we ran this SMIL page through a generator is raised the following flag:

```xml 
<?xml version="1.0" encoding="UTF-8"?>
<smil xmlns="http://www.w3.org/ns/SMIL" version="3.0" profile="http://www.idpf.org/epub/30/profile/content/">
                <body>
                                <seq id="id1" epub:textref="p010.xhtml" epub:type="chapter">
                                <par id="p1">
                                <text src="../pages/p010.xhtml#A-0"/>
                                <audio src="Pete_ILoveMyWhiteShoes.mp3" clipBegin="31.448829" clipEnd="37.892866"/></par>
                                </seq>
                </body>
</smil> 
``` 

“4:63The prefix "epub" for attribute "epub:textref" associated with an element type "seq" is not bound.”
 
We can flag these instances in the Kobo CMS but can anyone help us determine why ePubCheck wouldn’t be picking up this issue?
 
Thanks,
Ben Dugas

**Comment by mattgarrish** [comment]
From what I see, you're including media overlays in EPUB 2 files, where they're not recognized (the error about the attribute not being allowed shows attributes that are only allowed in the EPUB 2 package document).

I expect that's the reason why the SMIL files aren't being checked, as media overlays aren't part of EPUB 2.

(Note: I updated your comment as the pasted markup needed to be escaped.)

**Comment by bdugas** [comment]
Thanks @mattgarrish 
This makes sense as I suppose it's a case where our reading systems support ePub2 with SMIL. So it's not a redundant flag just one that's not going to result in the files having issue across our reading platforms.

**Comment by rdeltour** [comment]
@mattgarrish is right, closing as invalid.
